ambry-frontend/src/main/java/com/github/ambry/frontend/AmbrySecurityService.java

```diff
@@ -314,6 +320,12 @@ private void setGetBlobResponseHeaders(BlobInfo blobInfo, GetBlobOptions options
         restResponseChannel.setHeader("Content-Disposition", "attachment");
       }
     }
+    if (blobProperties.getContentEncoding() != null) {
+      restResponseChannel.setHeader(Headers.AMBRY_CONTENT_ENCODING, blobProperties.getContentEncoding());
```

Comment:
For a get blob response, this should be set as the HTTP standard `Content-Encoding` header instead of `x-ambry-content-encoding`

Reply:
Sure. Thanks for catching that, fixed it.